tests/v2api.test.ts

```diff
@@ -96,6 +110,10 @@ describe.runIf(semver.gte(nodeVersion, '18.13.0'))('V2 functions API', () => {
         }),
       })
 
+      for (const entry of files) {
+        expect(entry.runtimeAPIVersion).toBe(2)
+      }
```

Comment:
not related to the fix, but ensure that the `runtimeAPIVersion` returned by zipFunctions is 2.